core/templates/dev/head/collection_editor/CollectionEditor.js

```diff
@@ -104,3 +104,61 @@ oppia.controller('CollectionEditor', ['$scope',
         });
     };
   }]);
+
+oppia.controller('CollectionEditorNavbarBreadcrumb', ['$scope',
```

Comment:
This should go into its own file called 'CollectionEditorNavbarBreadcrumbController.js'. The collection editor is following a new design pattern which we're slowly propagating to the rest of Oppia's frontend.

I'm also not sure if this should be a controller or a directive. It seems to make more sense as a directive to me, since it's part of the collection editor page. WDYT?